data/completion/etelpmoc.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+# etelpmoc is the backwards half of complete: it de-serialises the tab
```

Comment:
maybe use the term `reverse`, I know it is backwards but I think it would read better here.